### PR TITLE
So/feature/carousel

### PR DESCRIPTION
**Pull request by DrSLMac**
# Pull Request Template
[ ] New feature (adds functionality)
-Add a carousel and a PR template

[ ] Describe the purpose of this PR (other) :

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes.
[X] I tested my changes in the browser
[ ] I tested with Cypress

# Checklist:
[ X] My code follows the style guidelines of this project
[X ] I have performed a self-review of my own code
[X ] I have commented my code, particularly in hard-to-understand areas
[ X] I have made corresponding changes to the documentation
[ ] My changes generate no new bugs
[ ] I have added tests that prove my fix is effective or that my feature works
[ ] New and existing unit tests pass locally with my changes
[ ] I have added screenshots, when necessary (see below) :

### CODE REVIEW

**Review by Maia-Cochran — APPROVED** [review]
This looks great, really excited to start adding data! :D Thank you for your work on this!

File: components/Carousel/CarouselItem.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ParallaxImage } from 'react-native-snap-carousel';
```
Comment (GOECHA) [inline components/Carousel/CarouselItem.js:2]: Parralax is great to view components inside the carousel. Great job getting the carousel up and running

File: components/Carousel/CarouselItem.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { View, Text, Pressable, SafeAreaView } from 'react-native';
+import styles from './styles';
+
+function CarouselItem({ item, index }, parallaxProps) {
+  return (
+    <Pressable onPress={() => alert('Image description:' + item.description)}>
```
Comment (GOECHA) [inline components/Carousel/CarouselItem.js:8]: I see thsi is where we can navigate to other areas, and to modal

File: components/Carousel/data.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const data = [
+    {
```
Comment (GOECHA) [inline components/Carousel/data.js:2]: dummy data

File: components/Carousel/styles.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { Dimensions, StyleSheet, Platform } from 'react-native';
+
+const { width: screenWidth } = Dimensions.get('window');
```
Comment (GOECHA) [inline components/Carousel/styles.js:3]: This separate styles sheet helps with the carousel and compartmentalizing the styles. I like how this is prganized.

File: components/Home.js [hunk 5]
```diff
@@ -1,12 +1,14 @@
...
 import Header from './Header'
 import GreetingButton from './GreetingButton'
+import Carousel from './Carousel/Carousel'
 
 const Home = () => {
     return (
         <View style={styles.homeContainer}>
             <Header />
             <GreetingButton />
+            <Carousel />
```
Comment (GOECHA) [inline components/Home.js:11]: Will the modal live here too??

File: pull-request-template.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# Pull Request Template
+[ ] New feature (adds functionality)
```
Comment (GOECHA) [inline pull-request-template.md:2]: This is awesome!

**Review by GOECHA — APPROVED** [review]
Great job with functionality, this looks clean and is very readable. I like the PR template to auto-populate. 
> 🟢 💚 😍 <

**Pull request merged**